pkg/asset/machines/machineconfig/osencryption.go

```diff
@@ -0,0 +1,71 @@
+package machineconfig
+
+import (
+	"fmt"
+
+	igntypes "github.com/coreos/ignition/config/v2_2/types"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/installer/pkg/asset/ignition"
+	"github.com/openshift/installer/pkg/types"
+)
+
+// ForOsEncryptionStrategy creates the MachineConfig based on EncryptionStrategy
+// This lays down the files to either disable encryption, or configure accordingly.
+func ForOsEncryptionStrategy(strategy types.EncryptionStrategy, role string) *mcfgv1.MachineConfig {
+	switch strategy {
+	case types.EncryptionStrategyDisabled:
+		return &mcfgv1.MachineConfig{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: "machineconfiguration.openshift.io/v1",
+				Kind:       "MachineConfig",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: fmt.Sprintf("50-%s-osencryption", role),
+				Labels: map[string]string{
+					"machineconfiguration.openshift.io/role": role,
+				},
+			},
+			Spec: mcfgv1.MachineConfigSpec{
+				Config: igntypes.Config{
+					Ignition: igntypes.Ignition{
+						Version: igntypes.MaxVersion.String(),
+					},
+					Storage: igntypes.Storage{
+						Files: []igntypes.File{
+							ignition.FileFromString("/etc/rhcos-no-clevis", "root", 0644, "\n"),
```

Comment:
@cgwalters is `"\n"` required or `""` works too?

Reply:
Note that since encryption is now disabled by default, this file is no longer read.

Reply:
Set null as the default, which acts just like Disabled in which case we do nothing.